package.json

```diff
@@ -29,7 +29,7 @@
     "luxon": "^3.5.0",
     "next": "^14.2.8",
     "next-sitemap": "^4.2.3",
-    "posthog-js": "^1.161.0",
+    "posthog-js": "latest",
```

Comment:
can we pin it to [1.203.0](https://github.com/PostHog/posthog-js/releases/tag/v1.203.0) exactly, without `^`?

I don't trust the package enough to use latest or `^`